components/help/help_controller.jsx

```diff
@@ -1,17 +1,18 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {Switch, Route} from 'react-router-dom';
 
-export default class HelpController extends React.Component {
-    static get propTypes() {
-        return {
-            children: PropTypes.node.isRequired
-        };
-    }
+import Messaging from './components/messaging';
+import Composing from './components/composing';
+import Mentioning from './components/mentioning';
+import Formatting from './components/formatting';
+import Attaching from './components/attaching';
+import Commands from './components/commands';
```

Comment:
Is there any particular reason that we have to use relative paths here? We seem to avoid them elsewhere in the webapp.

Reply:
No. We could use a path from the top level components directory. But I don't see why we shouldn't use a relative path. Doing a search for relative path imports gives me 423 results, so it's not exactly unique.

Reply:
Here is my two cent on this.  Use mostly absolute paths so any change of folder structure becomes slightly easy. 
Relative paths for anything in the same folder as most likely you will move them together. 

Ex: Container components and presentation components are usually in the same folder. So using relative path for importing presentation component into container component is perfectly fine but using a different component then, rather have an absolute path.

What do u guys think?